README.md

```diff
@@ -105,7 +105,9 @@ only when a rule or set of rules explicitly or implicitly permits it.
 Mutable Rules 
 -------------
 
-**201** Players may take their turn at any time, independent of other players. 
+**303** Players may take a turn at any time, independent of other players.
+Players are allowed to take any number of turns at the same time, regardless
+of any other ongoing turns taken by them.
 All players begin with zero points.
```

Comment:
I agree and think it would be best to keep it. At first glance it simply repeats the previous part of the sentence, but strictly speaking the "at the same time" could be interpreted as "opening PRs at the same time" thus invalidating opening a PR a minute after another.